dispatch/modules/podcasts/models.py

```diff
@@ -84,7 +87,13 @@ def __init__(self, *args, **kwargs):
     file = FileField(upload_to='podcasts/')
 
     def get_absolute_url(self):
-        return self.file.url
+        return "%spodcast/%s#%s" % (settings.BASE_URL, self.podcast.slug, self.get_slug_from_title())
```

Comment:
I think it would be best to remove this method and instead implement it as a helper in `ubyssey.ca`. Different dispatch themes will use different paths for podcasts, but this enforces a `podcasts/my-podcast/my-episode` pattern.

Same for `Podcast.get_absolute_url`

Reply:
The episodes do not have their own pages. So how would if I move the `get_absolute_url` function to `helpers.py`, how would i access it from the template?

Reply:
I will just zip the episode data together with the url in the `main.py`